components/breadcrumb/breadcrumb.spec.ts

```diff
@@ -463,3 +464,39 @@ export class NzTestBreadcrumbRtlComponent {
   @ViewChild(Dir) dir!: Dir;
   direction: Direction = 'rtl';
 }
+
+describe('breadcrumb', () => {
+  let fixture: ComponentFixture<NzBreadCrumbComponent>;
+  let component: NzBreadCrumbComponent;
+
+  beforeEach(() => {
+    TestBed.configureTestingModule({
+      imports: [NzBreadCrumbModule]
+    }).compileComponents();
+
+    fixture = TestBed.createComponent(NzBreadCrumbComponent);
```

Comment:
`NzBreadCrumbComponent` is a standalone component, no need to invoke `TestBed.configureTestingModule` or `TestBed.compileComponent`